scene/2d/light_2d.cpp

```diff
@@ -425,6 +425,17 @@ real_t PointLight2D::get_texture_scale() const {
 	return _scale;
 }
 
+#ifndef DISABLE_DEPRECATED
+bool PointLight2D::_set(const StringName &p_name, const Variant &p_value) {
+	if (p_name == "mode" && p_value.is_num()) { //Compatibility for Godot 3.
```

Comment:
I missed this earlier as I was reviewing on phone, there should be a space after `//`.
While changing it, might as well tweak language to match other similar 3.x compat checks.
```suggestion
	if (p_name == "mode" && p_value.is_num()) { // Compatibility with Godot 3.x.
```

Reply:
Done. Thanks.